tests/foreman/sys/test_hot_backup.py

```diff
@@ -14,42 +14,46 @@
 @CaseImportance: High
 
 @Upstream: No
-"""
 
-from robottelo.decorators import stubbed, tier3
+@Setup: Populate a Satellite with hosts/repos/contents
+"""
+from robottelo.constants import BACKUP_FILES
+from robottelo.decorators import stubbed, tier3, backup
+from robottelo.ssh import get_connection
 from robottelo.test import TestCase
 
 
+@backup
 class HotBackupTestCase(TestCase):
     """Implements ``katello-backup --online`` tests"""
 
-    @classmethod
-    def setUpClass(cls):
-        super(HotBackupTestCase, cls).setUpClass()
-
     @tier3
-    @stubbed()
-    def test_positive_directory_exists(self):
+    def test_positive_backup_with_existing_directory(self):
         """katello-backup with existing directory
 
         @id: 3f8285d5-c68b-4d84-8568-bcd7e4f7f19e
 
-        @Setup:
-
-        Populate a Satellite with hosts/repos/contents
-
         @Steps:
 
         1. Create a directory (ie /tmp/backup1)
         2. Run ``katello-backup /tmp/backup1``
-        3. Restore from backup
-
-        @Assert: The backup and restore succeed.
 
-        @CaseAutomation: notautomated
+        @Assert: The backup is successfully created in existing folder and
+        contains all the default files needed to restore.
 
-        @CaseLevel: System
         """
+        with get_connection() as connection:
+            connection.run('mkdir /tmp/backup1')
+            result = connection.run(
+                'katello-backup --online-backup /tmp/backup1', 'plain')
+            self.assertEqual(result.return_code, 0)
+            expected_msg = (
+                'BACKUP Complete, contents can be found in: /tmp/backup1'
```

Comment:
If you use a random name, then your expected message would have to be dynamic too.